templates/deployment.yaml

```diff
@@ -25,21 +25,72 @@ spec:
       serviceAccountName: {{ template "chart.serviceAccountName" . }}
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
-      priorityClassName: high-priority
```

Comment:
this is needed somehow, since it's eviction if a big problem. Maybe add to values?